.github/recipe.yaml

```diff
@@ -21,26 +23,25 @@ plugins:
   url_launcher: ["wearable-5.5", "tv-6.5"]
   wakelock: ["wearable-5.5"]
 
-  # Not supported by emulators.
-  camera: []
-  google_maps_flutter: []
-  network_info_plus: []
-  webview_flutter: []
-  webview_flutter_lwe: []
-
   # No tests.
   google_sign_in: []
   image_picker: []
   tizen_log: []
   tizen_notification: []
   wearable_rotary: []
 
-  # Permission cannot be granted without manual interaction.
+  # Functionality not available on emulator.
+  camera: []
   geolocator: []
-  permission_handler: []
+  network_info_plus: []
 
   # Only testable with the drive command: https://github.com/flutter-tizen/plugins/issues/272
   tizen_app_control: []
 
   # Test frequently breaks due to memory issue: https://github.com/flutter-tizen/plugins/issues/135
   video_player: []
+
+  # Temporarily disabled due to test_api error. Testable with the drive command.
+  # https://github.com/flutter-tizen/plugins/pull/397#issuecomment-1139299838
+  webview_flutter: []
+  webview_flutter_lwe: []
```

Comment:
I don't still understand why this error occurs. Any help?

Might be related to https://github.com/dart-lang/test/pull/1314?